src/libsystemd-network/sd-dhcp6-client.c

```diff
@@ -282,7 +282,7 @@ int sd_dhcp6_client_set_fqdn(
         return free_and_strdup(&client->fqdn, fqdn);
 }
 
-int sd_dhcp6_client_set_information_request(sd_dhcp6_client *client, int enabled) {
+int sd_dhcp6_client_set_information_request(sd_dhcp6_client *client, bool enabled) {
```

Comment:
We were not using `bool` in public (or intended-to-be-public-at-some-point) functions because it's not portable to C89/90. I think we should instead make the *other* functions use `int` instead.

Reply:
All booleans fixed.